File: Greedy/1_Change.java
import java.util.*;

public class Change {

	public static void main(String[] args) {
		Scanner sc = new Scanner(System.in);
		
		int n = sc.nextInt();
		int cnt = 0;
		int[]cointypes = {500, 100, 50, 10};
		
		for (int i = 0; i < 4; i++) {
			int coin = cointypes[i];
			cnt += n / coin;
			n %= coin;
		}
		System.out.println(cnt);
	}

}

File: Greedy/1_Change.c
#include <stdio.h>

int main(void)
{
	int i, n, cnt = 0;
	int cointypes[4] = {500, 100, 50, 10};
	int coin;
	
	scanf("%d", &n);
	for (i = 0; i < 4; i++) {
		coin = cointypes[i];
		cnt += n / coin;
		n %= coin;	
	}
	printf("%d", cnt);
}
